### DISCUSSION TITLE
After extracting audio and merging the same audio back in = audio stats dont look the same?

### DISCUSSION THREAD

**Discussion by giddyhara** [mifi/lossless-cut#1758] [Q&A]
so I did a test: Split the audio from the video, then add it back and see if it is the same.
To my surprise it was not the same:

![rrrr](https://github.com/mifi/lossless-cut/assets/128306269/d43c77c1-eeb6-4835-9bcd-7d5f6a4c7d8f)

The the audio track you see at the bottom is the original one I first extracted from this video file with losslesscut. And yet the duration is 1:46:05 and not 1.64:43. Also the audio codec is aac instead of aac mp4a

The weird part is when I add the 2 parts back together there is no desync... so I dont see much logic here. Can someone explain me why these difference are there and why there is no desync ????

**Comment by mifi** [comment]
My guess:
The audio i actually 1:46:05 but when it is together with the video in Your original file, it rounds the duration up to the video duration even though it is shorter. When you play back the original, probably there’s a short gap at the end of the video where the audio is silent

**Reply by giddyhara** [reply]
there is only 2 seconds at the end silence, rest is spoken from beginning to end. But what is a better way to do what I want to do? I want to split video and audio with lossless, then how can I combine the 2 again as if there were the original with lossless? (so same length, no desync)? Can I do this with lossless or is there an other tool to combine without encoding?

**Reply by mifi** [reply]
Yes you should be able to do it with losslesscut. You just have to try and then check the result if it sounds correct
